frontend/src/metabase/admin/permissions/selectors/confirmations.tsx

```diff
@@ -90,6 +90,24 @@ export function getPermissionWarning(
   return null;
 }
 
+export function getTableBlockWarning(
```

Comment:
nit: not just about table blocking

```suggestion
export function getBlockWarning(
```